core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/XMLPathTraversal.java

```diff
@@ -22,6 +24,7 @@
     private static final String XML_PATH = "xmlPath";
 
     private static final Pattern PATTERN = Pattern.compile("\\s*(?<" + TEXT + ">.+?)\\s*,\\s*(?<" + XML_PATH + ">.+)");
+    private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Don't need to instanciate Logger as @log4j2 annotation is already defined. 
Then use the variable `log` like ` log.error()`